install/docker.sh

```diff
@@ -1,15 +1,24 @@
-sudo apt install -y docker.io docker-buildx
+# Remove conflicting Docker packages
+sudo apt purge --auto-remove docker.io docker-buildx docker-doc docker-compose docker-compose-v2 podman-docker containerd runc
```

Comment:
Is it possible users would run this script in situations where they have an existing setup for any of these (for other purposes)?

It's correct that these can conflict with the official packages; just curious if this should either be detected first (are they installed?) and in that case ask the user to confirm, or depend on the package's "Conflicts:" metadata to make them fail if these are present.

Reply:
Fine to crash and burn on conflict.